ansible/roles/operator-pipeline/templates/openshift/pipelines/community-release-pipeline.yml

```diff
@@ -113,11 +113,9 @@ spec:
         - set-env-community
       params:
         - name: url
-          value: $(params.git_repo_url)
+          value: "$(params.git_repo_url)"
```

Comment:
Is this valid that this is utilizing a different parameter than in hosted pipeline?

Reply:
We don't have the PR branch URL as an input to the release pipeline; we could add it but, since this is called *after* the PR has been merged, the sha for the head of the PR will be present in the destination branch too.
I tested it and it seems to work fine.

Reply:
Makes sense, thanks for the clarification!